lib/index.js

```diff
@@ -10,51 +11,48 @@ var clone = require('clone');
  * Expose `Keen IO` integration.
  */
 
-var Keen = module.exports = integration('Keen IO')
+var KeenSegment = module.exports = integration('Keen IO')
   .global('Keen')
+  .global('KeenSegment')
   .option('ipAddon', false)
   .option('projectId', '')
-  .option('readKey', '')
+  .option('readKey', false)
```

Comment:
We can't make the default boolean here because our metadata service sets this option's type as a string. If we make it a boolean type, the settings will change into a checkbox and since we need to still grab the readkey string, you'll have to leave the default value as an empty string.